### ISSUE TITLE
Sub-title position - confusing and use should be constrained

### ISSUE THREAD

**Issue by paullenz** [mysociety/pombola#2001]
The sub-title field has a confusing title and, seemingly is only appropriate to use in the case of qualifications relating to educational establishments.  As a result we see, e.g. it being used for, in appropriately, adding in places.  Can it be re-named qualification and and be greyed out unless "education" has been selected as a category

![screenshot 2016-02-24 10 20 38](https://cloud.githubusercontent.com/assets/909811/13282354/81bfd870-dae0-11e5-9ecf-f1b2cc6aea6b.png)

**Comment by tmtmtmtm** [comment]
Agreed that it's confusing, but it is used for more then educational qualifications. It's also used to give more detail on people's job history roles (e.g. all the bits in brackets in that section of http://info.mzalendo.com/person/isaac-mwaura/experience/) — and, much more crucially, it's how we signify that someone is a Women's Representative in Mzalendo, or, in many (but not all) cases, that they were a Nominated rather than elected member (https://github.com/mysociety/pombola/issues/1984).
